src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -65,19 +65,22 @@ export class WorkspaceSavedObjectsClientWrapper {
 
     return [permission];
   }
-  private async validateSingleWorkspacePermissions(
-    workspaceId: string | undefined,
+  private async validateSingleObjectPermissions(
```

Comment:
It's out of the scope of this PR, but can we merge `validateSingleObjectPermissions` and `validateMultiObjectsPermissions`? To validate single object, we can just pass single `[object]` array, right? So one function which works for both cases.

For example,
```
private async validateObjectsPermissions(
    objects: Array<Pick<SavedObject, 'id' | 'type'>>,
    request: OpenSearchDashboardsRequest,
    permissionMode: WorkspacePermissionMode | WorkspacePermissionMode[]
) {}
```